public/director-movie-specific.php

```diff
@@ -0,0 +1,88 @@
+<?php
+include '../src/database/credentials.php';
+?>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>And Action</title>
+  <?php include "../templates/head.php"?>
+  <link rel="stylesheet" href="assets/css/director-movie-specific.css">
+</head>
+
+<body>
+  <!-- start navbar -->
+
+  <div class="navbar">
+    <?php include "../templates/navbar.php";?>
+  </div>
+
+  <!-- end navbar -->
+  <!-- start main container -->
+
+  <div class="container container-movie-specific">
+
+    <div class="text-wrapper">
+      <div class="description">
+        <h1>Narcos: Mexico</h1>
+        <p> 
+          lorem ipsum dolor sit amet lorem ipsum
+          lorem ipsum dolor sit amet lorem ipsum
+          lorem ipsum dolor sit amet lorem ipsum
+          lorem ipsum dolor sit amet lorem ipsum
+        </p>
+        <p class="movie-likes"> 
+          <a href="#">
+            <i class="fas fa-thumbs-up" style="font-size: 36px;"></i>
+          </a>
+          <span> 120.231.41 </span>
+        </p>
+      </div>
+      <div class="feedback">
+        <h2>Feedback</h2>
+        <?php
+          for ($comment = 0; $comment < 2; $comment++) {
+            echo "
+            <div class=\"comment-wrapper\"> 
+              <p> 
+              lorem ipsum dolor sit amet lorem ipsum
+              lorem ipsum dolor sit amet lorem ipsum
+              lorem ipsum dolor sit amet lorem ipsum
+              lorem ipsum dolor sit amet lorem ipsum
+              <br>
+              <span class=\"comment-author\"> 
+                <small>
+                
+                - user
+                
+                </small>
+              </span>
+              </p>
+              <div class=\"delete-comment\">
+                  <a href=\"#\">X</a>
+              </div> 
+            </div>";
+          }
+        ?>
+      </div>
+    </div>
+
+    <div class="movie-wrapper">
+      <a href="#">
+        <div class="movie">
+          <img src="https://m.media-amazon.com/images/M/MV5BZmFkMzc2NTctN2U1Ni00MzE5LWJmMzMtYWQ4NjQyY2MzYmM1XkEyXkFqcGdeQXVyNTIzOTk5ODM@._V1_.jpg" alt="movie-title">
+        </div>
+      </a>
+      <div class="form-delete-movie"> 
+        <form method="post"> 
+          <button class="delete-movie" type="submit" name="delete-movie"> Delete This movie </button> 
```

Comment:
Is het niet handiger op gewoon een <a> te gebruiken, dit is ook makkelijker te stylen. En je hebt dan ook geen form nodig want de url kan dan naar delete_movie.php waar die word verwijdert.

De styling komt ook niet helemaal overeen met het ontwerp
![afbeelding](https://user-images.githubusercontent.com/37366890/144226587-4c47f181-39fb-41fa-b82b-546a78926106.png)

Reply:
kleur is #DC3545